exercices/part2/practice2.py

```diff
@@ -0,0 +1,8 @@
+
+
+if __name__ == "__main__":
+    '''Write a program that prints out your name and waits for the user to press the Enter key before the 
+    program ends'''
+
+    print("Stan Andrei Alexandru")
+    input("Press Enter to exit")
```

Comment:
What happens if user doesn't press Enter or if presses something else??

Reply:
if the user press any other key, it will display that key on the line but to exit the program it must press the enter key